src/components/SingleVideoPage.js

```diff
@@ -1,28 +1,37 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { connect, useSelector } from "react-redux";
 import { useParams, useRouteMatch, withRouter, Link } from "react-router-dom";
-import { getVideos } from "../store/actions";
+import { getVidID } from "../store/actions";
```

Comment:
Probably a bad name, we're actually pulling the attributes for a whole video, so we should use something like getSingleVideo instead.